pkg/controller/virtualmachine/virtualmachine_controller.go

```diff
@@ -96,5 +96,6 @@ func (r *ReconcilePolicy) Reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{}, err
 	}
 
+	r.poolManager.MarkVMAsReady(fmt.Sprintf("%s/%s", request.Namespace, request.Name))
```

Comment:
maybe it would be nicer to pass namespace and name as arguments and join them inside the function